app/src/main/java/com/coveros/coverosmobileapp/website/MainActivity.java

```diff
@@ -26,29 +30,26 @@
     private String webName;
     private WebView browser;
     private AlertDialog dialog;
-
-    private ListView drawerList;
-    private static final String[] MENU_TITLES = new String[]{"Home","Blog","Bookmarks"};
+    private static final String TAG = "MainActivity";
+    private static final String[] MENU_TITLES = new String[]{"Home","Blog"};
     private DrawerLayout menu;
+    private ListView drawerList;
 
     public MainActivity(){
-        webName = "https://www.coveros.com/";
-    }
 
+        webName = "https://www3.dev.secureci.com";
```

Comment:
What happens when/if we did go to production? Are we going to have to change the code? This URL needs to be externalized so it can be injected by the build. Dev URL for testing, prod URL for release